.circleci/templates/setup.ps1

```diff
@@ -0,0 +1,6 @@
+Add-WindowsCapability -Online -Name OpenSSH.Server~~~~0.0.1.0
```

Comment:
let's rename this script to `.circleci/build-workbase-win/startup-template.ps1`.